packages/cli/src/index.ts

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+
+const { program } = require('commander')
+const { exec } = require('child_process');
+
+
+program
+    .command('version')
+    .description('output the version number')
+    .action(version)
+program
+    .command('[dbt_command]', { isDefault: true })
+    .description('runs dbt')
+    .action(dbt)
+
+
+
+function dbt () {
+
+    const args = process.argv.slice(2).join(' ')
+
+    exec(`dbt ${args}`, (err: string, stdout: string, stderr: string) => {
```

Comment:
does this pass the current environment to the dbt command?

Reply:
![image](https://user-images.githubusercontent.com/1983672/166716903-a5b761b2-5b6f-41f4-b022-67be0386a5ac.png)

![image](https://user-images.githubusercontent.com/1983672/166716928-d3a4f1ce-7ea2-4232-b67d-305063928181.png)

Yes, that's how it works by default, we could also specify extra variables by adding an extra argument to exec `{env:{} }`